conf/message

```diff
@@ -2,8 +2,8 @@ Ghost was successfully installed :)
 
 Next step:
 
-Create admin account at https://__DOMAIN____PATH_URL__/ghost
+Create admin account at https://$domain$path_url/ghost
```

Comment:
Not sure to understand that change ? In the install script, I see a ynh_replace_string for `__DOMAIN__` ... ? I doubt `$domain` correctly gets replaced because that file ain't run as a bash script ?